src/game/bg_public.h

```diff
@@ -704,13 +704,8 @@ typedef enum
 	PERS_RESPAWNS_PENALTY,         ///< how many respawns you have to sit through before respawning again
 
 	PERS_REVIVE_COUNT,
-	PERS_HEADSHOTS,                ///< Deprecated. Remove?
-	PERS_BLEH_3,
 
-	// mg42                        ///< TODO: I don't understand these here. Can someone explain?
-	PERS_HWEAPON_USE,              ///< enum 12 - don't change
-	// wolfkick
-	PERS_WOLFKICK
+	PERS_HWEAPON_USE = 12,         ///< non-zero when using a stationary hweapon (mg42=1, aagun=2)
```

Comment:
Just FYI this whole enum can hold max 16 values, and it's shared with the engine as it's part of playerstate, it's extremely important to keep track of this and not change the internal numbering as that causes incompatibility with old demos. I would in fact prefer to explicitly define all these 16 values, even the once that are spare and clearly mark them as unused.

Reply:
@Aciz please check out new revision